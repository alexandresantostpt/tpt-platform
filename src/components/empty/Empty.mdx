---
name: Empty
route: /empty
---
import { Playground, PropsTable } from 'docz'

import Empty from './Empty'

import '@cfg/scripts'
import '@cfg/style'

# Empty

<PropsTable of={ Empty }/>

## Basic usage

```
<Empty/>
```

<Playground>
    <Empty/>
</Playground>
