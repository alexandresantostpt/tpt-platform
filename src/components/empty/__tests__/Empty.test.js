import React from 'react'
import renderer from 'react-test-renderer'

import Empty from '@components/empty/Empty'

describe('Tests for Empty component', () => {
    it('Should rendering without crash ', () => {
        const tree = renderer.create(<Empty />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
