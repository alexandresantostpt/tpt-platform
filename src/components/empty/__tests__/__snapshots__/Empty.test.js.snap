// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Empty component Should rendering without crash  1`] = `null`;
